insights/config/specs.py

```diff
@@ -414,7 +414,7 @@
     "docker_images"             : "/usr/bin/docker images -q",
     "dumpdev"                   : "/bin/awk '/ext[234]/ { print $1; }' /proc/mounts",
     "getblockschedulers"        : "for device in $(ls /sys/block); do echo /sys/block/$device/queue/scheduler; done",
-    "httpd_pid"                 : "ps aux | grep httpd | grep -v grep | head -1 | awk '{print $2}'",
+    "httpd_pid"                 : "/bin/ps aux | grep /usr/sbin/httpd | grep -v grep | head -1 | awk '{print $2}'",
```

Comment:
Its not clear how this change is related to the other changes in this PR? Should this be a separate PR?

Reply:
@bfahr - This `httpd_pid` is the pre_command to get the PID of httpd for the `CommandSpec` of this parser.

Reply:
Got it, I should have paid attention to the line number.